src/EnergyPlus/InputProcessing/IdfParser.cc

```diff
@@ -269,14 +269,14 @@ json IdfParser::parse_idf(std::string_view idf, size_t &index, bool &success, js
             std::string name = fmt::format("{} {}", obj_name, s);
 
             if (!obj.is_null()) {
-                auto const &name_iter = obj.find("name");
+                auto const name_iter = obj.find("name");
```

Comment:
`find` returns an iterator (essentially a `std::pair`) by value, so this has to be a plain `auto`.